src/domain/hanke/hooks/useNavigateToApplicationList.ts

```diff
@@ -23,3 +23,15 @@ export default function useNavigateToApplicationList(hankeTunnus?: string) {
 
   return navigateToApplicationList;
 }
+
+function useNavigateToHankeList() {
+  const navigate = useNavigate();
+  const { HANKEPORTFOLIO } = useLocalizedRoutes();
+  function navigateToHankeList() {
+    navigate(HANKEPORTFOLIO.path, { state: { initiallyActiveTab: 4 } });
```

Comment:
That second parameter with state object to navigate function is not needed here as that is not used in hanke portfolio view. So that could be removed.

Reply:
Thanks, removed.